src/validator.cpp

```diff
@@ -16,22 +16,22 @@ limitations under the License.
 
 #include "libcellml/validator.h"
 
+#include <algorithm>
+#include <cmath>
+#include <libxml/uri.h>
+#include <stdexcept>
+
 #include "libcellml/component.h"
 #include "libcellml/importsource.h"
+#include "libcellml/model.h"
 #include "libcellml/reset.h"
+#include "libcellml/units.h"
 #include "libcellml/variable.h"
 #include "libcellml/when.h"
-
 #include "namespaces.h"
 #include "utilities.h"
 #include "xmldoc.h"
 
-#include <algorithm>
-#include <cmath>
-#include <stdexcept>
-
-#include <libxml/uri.h>
-
```

Comment:
Why reordering our header files using an order that is different from what was done in a recent suggested reordering?! So much inconsistency.

Reply:
I thought this was the consistent ordering defined by the style guide.  Associated header first, system headers, project headers.

Reply:
Am giving up. Will see that as part of issue #402.